libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/LedgerTypes/Tests.hs

```diff
@@ -180,4 +203,13 @@ utxoStateGen ::
 utxoStateGen =
   utxoStateSpec @era
     <$> genConwayFn @(PParams era) pparamsSpec
+    <*> genWitUniv @era 25
     <*> (lit <$> wff @(CertState era) @era)
+
+zzz :: IO ()
+zzz = do
+  univ <- generate $ genWitUniv @Shelley 25
+  m <- generate $ genFromSpec @ConwayFn (goodDrep @Shelley univ)
+  e <- generate $ genFromSpec @ConwayFn epochNoSpec
+  ans <- generate $ genFromSpec @ConwayFn (vstateSpec (eraWitUniv @Shelley 50) (lit e) (lit m))
+  putStrLn (show (prettyA ans))
```

Comment:
Could this be turned into a test and put into the testing suite? Otherwise it's just dead code.

Reply:
So I removed it. It was just a way to convince myself that things worked. I had failing tests, and this helped figure what was wrong. It is gone.